### PR TITLE
[framework] Admin dropdown UI & UX improvement

### PR DESCRIPTION
**Pull request by JanMolcik**
#### Description, the reason for the PR
To improve UI and UX, the "Domains filter" and "Go to Frontend" dropdown options list was adjusted.

#### Fixes issues
... 

#### License Agreement for contributions

- [x] I have read and signed [License Agreement for contributions](https://www.shopsys.com/license-agreement)

----
:globe_with_meridians: Live Preview:
  - https://jm-ssp-2944-admin-dropdown-ux.odin.shopsys.cloud
  - https://cz.jm-ssp-2944-admin-dropdown-ux.odin.shopsys.cloud

### CODE REVIEW

File: project-base/storefront/next-env.d.ts [hunk 1]
```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
```
Comment (chlebektomas) [inline project-base/storefront/next-env.d.ts]: I think we do not need this :)

File: packages/framework/src/Resources/translations/messages.cs.po [hunk 2]
```diff
@@ -3514,15 +3514,9 @@ msgstr "Vyhledávání hledá v názvu produktu, katalogovém čísle produktu,
 msgid "Searching searches in slugs. It is possible to use wildcards * and ?."
 msgstr "Vyhledávání hledá ve sluzích. Je možné používat operátory * a ?."
 
-msgid "Select all"
```
Comment (chlebektomas) [inline packages/framework/src/Resources/translations/messages.cs.po:3544]: Do we want to delete these translations?

File: packages/framework/assets/styles/admin/components/list/domain.less [hunk 3]
```diff
@@ -60,3 +60,46 @@
...
+        max-width: 335px;
+        overflow-y: auto;
+        display: flex;
+        flex-direction: column;
+        grid-gap: 0;
+        background-color: white;
+        padding: 0;
+
+        .form-choice {
+            padding: 10px 1em;
```
Comment (chlebektomas) [inline packages/framework/assets/styles/admin/components/list/domain.less]: I am just curious if we are mixing these units.

**Review by chlebektomas — CHANGES_REQUESTED** [review]
Please, check the standarts as well.

File: packages/framework/src/Resources/translations/messages.en.po [hunk 4]
```diff
@@ -251,7 +251,7 @@ msgid "Adverts - view"
 msgstr ""
 
 msgid "All - full"
-msgstr ""
+msgstr "All"
```
Comment (vitek-rostislav) [inline packages/framework/src/Resources/translations/messages.en.po:257]: there should not be any translation for the `en` locale

File: packages/framework/src/Resources/views/Admin/Form/adminDomainsForm.html.twig [hunk 5]
```diff
@@ -11,10 +11,12 @@
...
     </ul>
-    <div class="select-domain-buttons">
-        <button class="btn btn--light btn--small js-domains-select-all" type="button">{{ 'Select all'|trans }}</button>
-        <button class="btn btn--light btn--small js-domains-select-none"
-                type="button">{{ 'Select none'|trans }}</button>
+    <div class="domain-filter__buttons">
+        <label class="checkbox">
+            <input type="checkbox" class="checkbox__input js-domains-select-all-checkbox">
+            <span class="checkbox__checkmark"></span>
+            <span class="checkbox__label">{{ 'All - full'|trans }}</span>
```
Comment (vitek-rostislav) [inline packages/framework/src/Resources/views/Admin/Form/adminDomainsForm.html.twig]: I think you need another translation key here - just "All" probably?

**Review by vitek-rostislav — CHANGES_REQUESTED** [review]
Hi, for testing purposes, could you also add a [DROP] commit with multiple domains set?

**Pull request merged**